.circleci/config.yml

```diff
@@ -22,6 +24,68 @@ jobs:
           path: build/test-results
       - store_artifacts:
           path: build/reports/tests
+      - persist_to_workspace:
+          root: .
+          paths:
+            - build/libs
+            - build.gradle.kts
+
+  integration_tests:
+    docker:
+      - image: cimg/openjdk:11.0
+        environment:
+          _JAVA_OPTIONS: -Xmx1024m -XX:ParallelGCThreads=2 -XX:ConcGCThreads=2 -XX:ParallelGCThreads=2 -Djava.util.concurrent.ForkJoinPool.common.parallelism=2 -Dorg.gradle.daemon=false
+      - image: quay.io/hmpps/hmpps-auth:latest
+        environment:
+          - SERVER_PORT=9090
+          - SPRING_PROFILES_ACTIVE=dev
+      - image: postgres:10.16
+        environment:
+          POSTGRES_DB: jpc
+          POSTGRES_USER: jpc
+          POSTGRES_PASSWORD: letmein
+        command: -p 5433
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/app
+      - run:
+          name: Wait for auth
+          command: dockerize -wait http://localhost:9090/auth/health -timeout 1m
+      - run:
+          name: Get wiremock
+          command: curl -o wiremock.jar https://repo1.maven.org/maven2/com/github/tomakehurst/wiremock-standalone/2.27.2/wiremock-standalone-2.27.2.jar
+      - run:
+          name: Run wiremock
+          command: java -jar wiremock.jar --port 9999
+          background: true
+      - run:
+          name: Run the java app.
+          command: env spring.profiles.active=dev java -Xmx1024m -jar ~/app/build/libs/calculate-journey-variable-payments-*.jar
```

Comment:
Not 100% sure why I need to specify `~/app/` here for jar execution as the folder is added at the top of the steps: above.